library/x509_crl.c

```diff
@@ -511,30 +511,30 @@ int mbedtls_x509_crl_parse( mbedtls_x509_crl *chain, const unsigned char *buf, s
                                "-----END X509 CRL-----",
                                buf, NULL, 0, &use_len );
 
-        if( ret == 0 )
```

Comment:
I think this block was correctly indented, but the previous `if` branch was wrong.

Reply:
Indeed, thanks for noticing. Given that there was a single commit, I amended it and force-pushed it.